<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BlackJack</class>
 <widget class="QWidget" name="BlackJack">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>655</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BlackJack</string>
  </property>
  <widget class="QPushButton" name="hitButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>460</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>31</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Hit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stayButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>530</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>31</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Stay</string>
   </property>
  </widget>
  <widget class="QLabel" name="playerLabel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>31</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Player</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>751</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="dc_0">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../VSC/CPP/Blackjack/Card PNGs/1S.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="dc_1">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../VSC/CPP/Blackjack/Card PNGs/1S.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="dc_2">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../VSC/CPP/Blackjack/Card PNGs/1S.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="dc_3">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../VSC/CPP/Blackjack/Card PNGs/1S.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="dc_4">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../VSC/CPP/Blackjack/Card PNGs/1S.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="dc_5">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../VSC/CPP/Blackjack/Card PNGs/1S.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="dc_6">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../VSC/CPP/Blackjack/Card PNGs/1S.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="dealerLabel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>240</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>31</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Dealer</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>751</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="pc_0">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../VSC/CPP/Blackjack/Card PNGs/1S.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="pc_1">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../VSC/CPP/Blackjack/Card PNGs/1S.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="pc_2">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../VSC/CPP/Blackjack/Card PNGs/1S.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="pc_3">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../VSC/CPP/Blackjack/Card PNGs/1S.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="pc_4">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../VSC/CPP/Blackjack/Card PNGs/1S.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="pc_5">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../VSC/CPP/Blackjack/Card PNGs/1S.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="pc_6">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../VSC/CPP/Blackjack/Card PNGs/1S.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>600</y>
     <width>611</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Win:      Loss:     Push:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
